tests/bluetooth/bsim_bt/bsim_test_l2cap_stress/prj.conf

```diff
@@ -18,7 +18,14 @@ CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=n
 # L2CAP MPS
 # 23+27+27=77 makes exactly three full packets
 CONFIG_BT_L2CAP_TX_MTU=77
-CONFIG_BT_BUF_ACL_TX_SIZE=77
+
+# Use this to send L2CAP PDUs without any fragmentation.
+# In this particular case, we prefer fragmenting to test that code path.
+# CONFIG_BT_BUF_ACL_TX_SIZE=81
```

Comment:
Why is this changed from 77 to 81? Header size not taken into account before?

Reply:
yep